content/docs/07-reference/ai-sdk-ui/01-use-chat.mdx

```diff
@@ -274,7 +381,8 @@ Allows you to easily create a conversational user interface for your chatbot app
       description:
         'React and SolidJS only. Function to add a tool result to the chat. This will update the chat messages with the tool result and call the API route if all tool results for the last message are available.',
     },
-  ]}
+
+]}
```

Comment:
this is usually an indication that the formatting is off somewhere

Reply:
fixed formatting!